indico/modules/events/timetable/client/js/preprocess.ts

```diff
@@ -165,7 +167,6 @@ export function preprocessTimetableEntries(
     dayEntries,
     unscheduled: (eventInfo.contributions || []).map(c => ({
       type: 'contrib',
-      id: c.uniqueId,
```

Comment:
I think we still need to keep some kind of id for the unscheduled contribs. `getEntryUniqueId` should work here.